index.bs

```diff
@@ -8537,6 +9372,8 @@ Operations present in other neural network inference APIs can often be emulated
 
 <p class="note">{{Float16Array}} is at <a href="https://tc39.es/process-document/">ECMA Stage 3</a> signaling its design is finished. Implementers wanting to enable this type ahead native implementations can emulate the type by passing raw bits via {{Uint16Array}}. <a href="https://github.com/webmachinelearning/webnn/issues/373">[Issue webnn#373]</a></p>
 
+<p class="note">There is no Uint4Array/Int4Array class. Nybbles are stored in byte arrays of {{Uint8Array}} with the lower nybble in the lower bits, meaning tensor element 0 would be found in byte 0 bits 0-3, and tensor element 1 in byte 0 bits 4-7 (and so on, with tensor element 5 in byte 2 bits 4-7). Odd tensor element counts are rounded up to whole bytes, and last nybble is ignored, meaning a 5 element tensor uses 3 bytes.</a></p>
```

Comment:
I think "nibble" is the preferred spelling. 

Also, the examples may be a bit confusing, as the first example mentions tensor element 5 (so, a 6-or-more element tensor) which would pack the that element's bits into the upper nibble, whereas the second example mentions a 5 element tensor which would pack the last element's bits into the lower nibble. Maybe intentionally pick a number other than 5 for the first or second example?

Reply:
The definitions of [byte length](https://www.w3.org/TR/webnn/#mloperanddescriptor-byte-length) and [validate buffer with descriptor](https://www.w3.org/TR/webnn/#validate-buffer-with-descriptor) should get updated. Probably more places too.

Reply:
Add to `MLOperandDataType` enum too.

Reply:
Also the definition of the `cast()` operator and the `cast` algorithm - even just to call out that uint4 is not permitted.